astropy/convolution/kernels.py

```diff
@@ -97,8 +97,12 @@ class Gaussian2DKernel(Kernel2D):
 
     Parameters
     ----------
-    stddev : number
-        Standard deviation of the Gaussian kernel.
+    x_stddev : float
+        Standard deviation of the Gaussian in x.
+    y_stddev : float
+        Standard deviation of the Gaussian in y.
+    theta : float
+        Rotation angle in radians. Note: increases clockwise.
```

Comment:
That sounds odd with "increases clockwise"...

Reply:
Ops, should that be counter clockwise like Gaussian2D model?

Reply:
Probably, but the parameter description there is also a bit "better": "The rotation angle increases counterclockwise."

Reply:
@pllim The rotation angle increases *counterclockwise*.

Reply:
Yes, my bad. I didn't proof read the doc. I'll fix that in my next commit.